prac_06/programming_language.py

```diff
@@ -0,0 +1,31 @@
+"""
+Programming Languages.
+Estimate: 20 mins. Start: 4:59pm
+Actual: 32 mins. End: 5:31pm
+"""
+
+
+class ProgrammingLanguage():
+    """Programming Language class"""
+
+    def __init__(self, field, typing, reflection, year):
+        """Initialise Programming Language object."""
+        self.field = field
```

Comment:
Would "name" would be a better name for the names of the programming languages than "field" ?

Reply:
That's a great point. Field is the name so that would make it much easier to understand on first glance if I just call it 'name'.